sources/packages/web/src/views/aest/institution/Profile.vue

```diff
@@ -3,8 +3,13 @@
     <div class="mx-5 py-4">
       <body-header title="Profile" class="m-1">
         <template #actions>
-          <v-btn text :color="COLOR_BLUE" @click="editProfile"
-            ><font-awesome-icon :icon="['fas', 'cog']" class="mr-2" />Edit
+          <v-btn
+            class="float-right"
+            text
+            @click="editProfile"
+            variant="title"
+            :color="COLOR_BLUE"
+            ><v-icon icon="fa:fa fa-cog" class="mr-1" /> Edit
```

Comment:
The `v-btn` expose `prepend-icon` and `append-icon`. Can we please use these properties instead of using the `v-icon`?

Reply:
Ya... icon for v-btn , i will fix in my next PR